nix/module/worker.nix

```
@@ -0,0 +1,116 @@
+{ pkgs, lib, config, ... }:
+
+{
+  options.services.sioworker = {
+    enable = lib.mkEnableOption "sioworker";
+
+    package = lib.mkPackageOption pkgs "sioworkers" {
+      default = [ "python38Packages" "sioworkers" ];
+    };
+
+    filetrackerUrl = lib.mkOption {
+      default =
+        if ((builtins.hasAttr "filetracker" config.services) && config.services.filetracker.enable) then
+          "http://${config.services.filetracker.listenAddress}:${builtins.toString config.services.filetracker.port}" else null;
+      description = "The filetracker URL for the sioworker to connect to";
+      type = lib.types.str;
+    };
+
+    filetrackerCache = lib.mkOption {
+      default = "/var/cache/sioworker/filetracker";
+      description = lib.mdDoc ''
```

```suggestion
      description = ''
```

Lets be consistent with the use of mdDoc here, we will never make use of it since it's a nixpkgs thing so removing it seems ok.